pipeline/docker/__init__.py

```diff
@@ -110,5 +129,26 @@ def create_docker_compose(path, **environment_vars):
             },
         },
     }
+
+    if gpu_index is not None:
```

Comment:
it looks like the value of `gpu_index` is never used?

Reply:
fixed, good catch